gravity-info-server/src/main.rs

```diff
@@ -98,18 +105,80 @@ async fn get_bridge_volume() -> impl Responder {
     }
 }
 
+#[get("/transactions/send_to_eth")]
```

Comment:
this looks good, I think the question now is how much processing to do server side and how much to do frontend side. Sending the tx will allow for the graph to be reordered/changed entirely client side which I think is good as far as being responsive if you're up to having the complexity on that end.